src/explorer/SubscriptionTreeItem.ts

```diff
@@ -101,31 +98,14 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
         context.showCreatingTreeItem(nonNullProp(wizardContext, 'newSiteName'));
 
-        if (!advancedCreation) {
+        if (!context.advancedCreation) {
             await setDefaultRgAndPlanName(wizardContext, siteStep);
         }
 
-        try {
-            await wizard.execute();
-        } catch (error) {
-            // if there is an error when creating, prompt the user to try advanced creation
-            // tslint:disable-next-line: strict-boolean-expressions
-            if (!parseError(error).isUserCancelledError && !advancedCreation) {
-                const message: string = `Modify the setting "${extensionPrefix}.${configurationSettings.advancedCreation}" if you want to change the default values when creating a Web App in Azure.`;
-
-                // tslint:disable-next-line: no-floating-promises
-                ext.ui.showWarningMessage(message, AppServiceDialogResponses.turnOnAdvancedCreation).then(async result => {
-                    if (result === AppServiceDialogResponses.turnOnAdvancedCreation) {
-                        await workspaceConfig.update('advancedCreation', true, ConfigurationTarget.Global);
-                    }
-                });
-            }
-            throw error;
-        }
-
+        await wizard.execute();
         context.telemetry.properties.os = wizardContext.newSiteOS;
         context.telemetry.properties.runtime = wizardContext.newSiteRuntime;
-        context.telemetry.properties.advancedCreation = advancedCreation ? 'true' : 'false';
+        context.telemetry.properties.advancedCreation = context.advancedCreation ? 'true' : 'false';
```

Comment:
You can remove this line. It's already in the shared package: https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/treeDataProvider/AzExtParentTreeItem.ts#L67